Makefile

```diff
@@ -1,82 +1,97 @@
 all:
-	@echo "Make All"
+	@echo "make all not implemented"
 
-build:
-	cargo build
-node:
-	cargo build --package $(call pkgid, litentry-collator)
+## variant declaration
 
-test-node:
-	cargo test --package $(call pkgid, litentry-collator)
+NODE_BIN=litentry-collator
+RUNTIME=litentry-parachain-runtime
 
-test:
-	cargo test
+## build release
+
+.PHONY: build-all
+build-all:
+	cargo build --release
+
+.PHONY: build-node
+build-node:
+	cargo build -p $(call pkgid, $(NODE_BIN)) --release
+
+# TODO: use srtool to build wasm
+.PHONY: build-runtime
+build-runtime:
+	cargo build -p $(call pkgid, $(RUNTIME)) --release
 
-# benchmark build
+.PHONY: build-docker
+build-docker:
+	@./scripts/build-docker.sh
+
+.PHONY: build-spec-dev
+build-spec-dev:
+	./target/release/$(NODE_BIN) build-spec --chain dev --disable-default-bootnode > ./source/local.json
+
+.PHONY: build-benchmark
 build-benchmark:
 	cargo build --features runtime-benchmarks --release
 
-build-spec:
-	./target/release/litentry-collator build-spec --chain dev --disable-default-bootnode > ./source/local.json
+## test
+
+.PHONY: test-all
+test-all:
+	cargo test
```

Comment:
The test-all here sounds a bit confusing. I expected to run all available tests here. Also, do we need to do cargo test on parachain?